domain/src/main/java/org/oppia/android/domain/topic/StoryProgressTestHelper.kt

```diff
@@ -231,6 +231,27 @@ class StoryProgressTestHelper @Inject constructor(
     )
   }
 
+  /**
+   * Marks full topic progress on the second test topic for a particular profile.
+   *
+   * @param profileId The profile we are setting topic progress for.
+   * @param timestampOlderThanOneWeek If the timestamp for completing the topic is from more than one week ago.
```

Comment:
```suggestion
   * @param profileId the profile we are setting topic progress for
   * @param timestampOlderThanOneWeek if the timestamp for completing the topic is from more than one week ago
```

A nit, but this is more conformant with the style guide.